tequilapi/endpoints/identities_test.go

```diff
@@ -1,15 +1,108 @@
 package endpoints
 
 import (
+	"net/http"
 	"net/http/httptest"
 	"testing"
 
+	"bytes"
 	"github.com/mysterium/node/identity"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestListIdentities(t *testing.T) {
+var identityUrl = "/identities/123/registration"
```

Comment:
- It could be constant
- Dont mislead with wrong URL . I would put just `""` or `"/test"`